src/conv/heur/gfx908_metadata.model

```diff
@@ -0,0 +1,317 @@
+                    "FilterDim0": 1.0000308752059937,
+                    "FilterDim1": 2.1867706775665283,
+                    "FilterDim2": 2.1919846534729004,
+                    "OutChannels": 315.09002685546875,
+                    "OutDepth": 1.0008162260055542,
+                    "OutHeight": 191.78111267089844,
+                    "OutWidth": 203.95440673828125,
+                    "BatchSize": 19.30280303955078,
+                    "Padding0": 0.9998764991760254,
+                    "Padding1": 0.5854535102844238,
+                    "Padding2": 0.5849190354347229,
+                    "Stride0": 1.0000327825546265,
+                    "Stride1": 1.3545188903808594,
+                    "Stride2": 1.3545188903808594,
+                    "Dilation1": 1.0090287923812866,
+                    "Dilation2": 1.0090287923812866,
+                    "Layout": 0.0001389347162330523,
+                    "Precision": 0.08208919316530228,
+                    "Direction": 0.9971132278442383,
+                    "GroupSize": 1.3931447267532349
+                },
+                "std": {
+                    "InChannels": 391.0542907714844,
+                    "InDepth": 0.08177872747182846,
+                    "InHeight": 181.0471954345703,
+                    "InWidth": 191.44808959960938,
+                    "FilterDim0": 0.007857984863221645,
+                    "FilterDim1": 1.8559739589691162,
+                    "FilterDim2": 1.8844841718673706,
+                    "OutChannels": 388.1698303222656,
+                    "OutDepth": 0.10612594336271286,
+                    "OutHeight": 218.2330780029297,
+                    "OutWidth": 230.737060546875,
+                    "BatchSize": 95.23401641845703,
+                    "Padding0": 0.011112269014120102,
+                    "Padding1": 1.0408027172088623,
+                    "Padding2": 1.0403735637664795,
+                    "Stride0": 0.00572739215567708,
+                    "Stride1": 0.4816601276397705,
+                    "Stride2": 0.4816601276397705,
+                    "Dilation1": 0.48016685247421265,
+                    "Dilation2": 0.48016685247421265,
+                    "Layout": 0.011786249466240406,
+                    "Precision": 0.3345409035682678,
+                    "Direction": 0.8201566934585571,
+                    "GroupSize": 8.84158992767334
+                }
+            },
+            "gt": {
+                "mean": {
+                    "algorithm": 3.8200466632843018,
+                    "solver": 2.8911001682281494,
+                    "solverTime": 1.5078306198120117
+                },
+                "std": {
+                    "algorithm": 2.559572696685791,
+                    "solver": 2.3032851219177246,
+                    "solverTime": 109.3656005859375
+                }
+            }
+        }
+    },
+    "model_name": "TunaNetv1",
+    "num_inputs": 24,
+    "num_outputs": 18
+}
```

Comment:
It is better to always add a newline at the end of file

Reply:
Done.